Tests/DependencyInjection/Compiler/AnnotationCompilerPassTest.php

```diff
@@ -90,6 +90,32 @@ public function testMinimalAdmin()
         );
     }
 
+    public function testIdForAdmin()
+    {
+        /*
+         * @Admin(class="Sonata\AdminBundle\Entity\Foo", id="my.id")
+         */
+        $annotation = new Admin();
+        $annotation->class = 'Sonata\AdminBundle\Entity\Foo';
+
+        $meta = new ClassMetadata('Sonata\AdminBundle\Tests\Fixtures\Entity\Foo');
+
+        $annotation->processMetadata($meta);
+
+        $this->assertSame(
+            $meta->tags['sonata.admin'][0],
+            array(
+                'manager_type' => 'orm',
+                'id' => 'my.id',
+                'group' => 'Admin',
+                'label' => 'Foo',
+                'show_in_dashboard' => true,
+                'keep_open' => false,
+                'on_top' => false,
+            )
+        );
+    }
+
```

Comment:
There is no need to create another test (which tests again default values like `manager_type`, `group`, etc...). Please add the test into the `testAdmin()` method.